### DISCUSSION TITLE
Shards failing(field limit problem)

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#3479] [Unsupported Versions]
Hello, I am using Security onion 2.3.33, and am running into an issue as seen by the image at the bottom(not the first image). According to the documentation, you can change this field limit value via the following command: 
curl -XPUT -H'Content-Type: application/json' localhost:9200/logstash-syslog-*/_settings -d'{ "index.mapping.total_fields.limit": 2000 }
The following error occurs(Note, i changed the filepath for testing purposes, the original code still did not work)
![image](https://user-images.githubusercontent.com/27656903/111425184-2ad3bc80-86c9-11eb-895b-ab87207075ac.png)

In my case, I wanted to make this a persistent change, but any update I make to the template files gets removed after restarting logstash.

![image](https://user-images.githubusercontent.com/27656903/111424891-c153ae00-86c8-11eb-8b81-8c3b31dabd9b.png)

Any help would be much appreciated

**Comment by weslambert** [comment]
I would try to limit the number of fields by not dynamically parsing, and/or ensure you are modifying templates in the local vs default `/opt/so/saltstack/<insert>/salt/elasticsearch/templates/so` directory
